packages/abap-deploy-config-sub-generator/src/app/index.ts

```diff
@@ -86,10 +85,6 @@ export default class extends DeploymentGenerator {
             watchTelemetrySettingStore: false
         });
 
-        if ((this.env as unknown as YeomanEnvironment).conflicter) {
-            (this.env as unknown as YeomanEnvironment).conflicter.force = this.options.force ?? true;
```

Comment:
No longer required as writing phase is in the correct place and yeoman does not complain about over writing files